core/PRP/Camera/plCameraBrain.h

```diff
@@ -79,6 +79,38 @@ class PLASMA_DLL plCameraBrain1 : public virtual plCameraBrain {
     virtual void read(hsStream* S, plResManager* mgr);
     virtual void write(hsStream* S, plResManager* mgr);
 
+	float getVelocity() const { return fVelocity; }
+	float getAcceleration() const { return fAccel; }
+	float getDeceleration() const { return fDecel; }
+	float getPOAVelocity() const { return fPOAVelocity; }
+	float getPOAAcceleration() const { return fPOAAccel; }
+	float getPOADeceleration() const { return fPOADecel; }
+	hsVector3 getPOAOffset() const { return fPOAOffset; }
+	float getXPanLimit() const { return fXPanLimit; }
+	float getZPanLimit() const { return fZPanLimit; }
+	float getPanSpeed() const { return fPanSpeed; }
+	float getZoomRate() const { return fZoomRate; }
+	float getZoomMax() const { return fZoomMax; }
+	float getZoomMin() const { return fZoomMin; }
+	hsBitVector getFlags() const { return fFlags; }
```

Comment:
I'm not sure I fully understand, getFlags() should return a non-const reference? Should getPOAOffset work the same?